test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.Customer.Index.html

```diff
@@ -0,0 +1,42 @@
+﻿
```

Comment:
this and next file names do not look right.  suggest .../MvcTagHelpersWebSite.Customer.Index.html (for the file below) and .../MvcTagHelpersWebSite.Customer.Index.Errors.html (for this file)